avocado/plugins/spawners/podman.py

```diff
@@ -58,6 +79,10 @@ async def spawn_task(self, runtime_task):
         entry_point_args = task.get_command_args()
         entry_point_args.insert(0, "task-run")
         entry_point_args.insert(0, entry_point_cmd)
+        if task.runnable.kind == "avocado-instrumented":
```

Comment:
I don't think spawners should have to know about task kinds... the information that any spawner need should be generalized.  So, if "avocado-instrumented" tests have a given characteristic that requires an extra handling, these should be generalized to a concept that can be applied to other task/runnable kinds.

This is the idea behind the "core/avocado" requirement.  Other task kinds may have the same requirement, not being able to run unless avocado is present on the environment that will actually run the task.

Reply:
Yep, I agree, as I mentioned in another comment, don't worry about the `podman.py` file, I did it to make things easy on my side. Added it here just for testing purposes, but those changes are not supposed to get merged and don't make much sense as you stated.